docs/UserGuide.adoc

```diff
@@ -111,6 +111,26 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+== Deleting multiple persons : `deletemulti`
+
+Deletes the specified list of persons from the address book. Irreversible. +
+Format: `deletemulti INDEX`
```

Comment:
Could change to deletemulti INDEX [INDEX...]
so it's more clear that it accepts multi param in the user guide

Reply:
Thanks for suggesting that, was wondering how to show that.